### PR TITLE
feat: AppKit Solana Example

### PR DESCRIPTION
**Pull request by rtomas**
+ Simple example to show how to integrate AppKit + Solana

### CODE REVIEW

File: dapps/web3modal/react-solana/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    }
+
+    const handleSign = async () => {
+      if (!walletProvider || !address) {
+          printConsole('walletProvider or address is undefined');
+          return;
+      }
+
+      const encodedMessage = new TextEncoder().encode('Appkit Solana')
+      const signature = await walletProvider.signMessage(encodedMessage) as Uint8Array
```
Comment (zoruka) [inline dapps/web3modal/react-solana/src/App.tsx:96]: is this needed?
```suggestion
      const signature = await walletProvider.signMessage(encodedMessage)
```

File: dapps/web3modal/react-solana/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+      const joinSignatureValue = signature.join('');
```
Comment (zoruka) [inline dapps/web3modal/react-solana/src/App.tsx:97]: maybe would be clearer if we use base58 to encode this

**Review by zoruka — COMMENTED** [review]
Could we have this deployed also?

**Review by glitch-txs — APPROVED** [review]

**Review by Johnson1s — COMMENTED** [review]
- My email is john@doe.com - 1724971505743
- JSON-RPC Request Approved
method
personal_sign
address
0xd689dfbd63db7e1f54c95b9d8147ff05cbc76c3b
valid
true
result
0xd2bfdf2e77f6dc1b9e3f694fbe9390de1e973e1e355b37c01c977de53d7ba2c849739b72f8a73921e27d99122c24123c78a6616596270eb1b2d5bb399ad334a21c

**Pull request merged**